.gitlab-ci.yml

```diff
@@ -48,6 +48,8 @@ tox:
     - java -version
     - pip install 'tox < 4'
     - tox -e py
+  except:
+    - tags
```

Comment:
I may be missing some context here, is this skipping tests for tags?

Reply:
yes, because tag step is only pulling existing docker image, tag it and push it to nexus.
not building any new docker images.
The tests already run while we building docker images.